src/components/Blog/Post/EditPost.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Modal, Button } from 'react-bootstrap';
+import { Formik, Form, ErrorMessage, Field } from 'formik';
+
+import TextError from '../../UI/FormError/FormError';
+import './EditPost.css';
+import Spinner from '../../UI/Spinner/Spinner';
+
+function EditPost(props) {
+
+  const initialValues = {
+    title  : props.post ? props.post.title  : '',
+    content: props.post ? props.post.content: ''
+  };
+
+  const onSubmit = values => {
+    props.editpost(values.title, values.content, props.post.id, props.post.key, 'PUT');
```

Comment:
Edit post on details post page is not rendering updated data immediately. It's working from the blog list page